skymap_scanner/server/start_scan.py

```diff
@@ -114,7 +114,7 @@ def __init__(
         self.reporter_start_time = 0.0
 
         self._call_order = {
-            'current_previous': {  # current_fucntion: previous_fucntion(s)
+            "current_previous": {  # current_fucntion: previous_fucntion(s)
```

Comment:
I see you've started using Black :smile: One thing I've done to keep quote-conversions down is use `--skip-string-normalization` "smartly": `black $(if [[ $(grep -o -P '#\\s*fmt:\\s*quotes-ok*' $FILE) && $(grep -o -P '\\x27[^\\x27]+\\x27' $FILE | wc -l) -gt 10 ]]; then echo '--skip-string-normalization'; fi) $FILE`. This adds the benefit of keeping `git blame` true to the author but also doesn't fully exclude quote-conversions in newer/smaller files.

Reply:
Yes, sorry, I have it run automatically on save. This file can simply be reverted.

Reply:
Are you using Sublime? If so, I can give you some on-save code that does this automatically